pkg/controller/v1beta1/inferenceservice/reconcilers/raw/raw_kube_reconciler.go

```diff
@@ -64,10 +65,14 @@ func NewRawKubeReconciler(client client.Client,
 	if workerPodSpec != nil {
 		multiNodeEnabled = true
 	}
+	depl, err := deployment.NewDeploymentReconciler(client, clientset, scheme, componentMeta, workerComponentMeta, componentExt, podSpec, workerPodSpec)
```

Comment:
why pass client and clientSet?
imho we should rely on clientSet.

Reply:
don't want to change too much of the existing code, didn't touch the `client` usage. The `clientset` addition was needed to fetch the inferenceservice-config. Kserve doesn't have watchers for configmaps so I had to use the clientset to fetch the CM rather than re-using the client.

Reply:
I guess I understand why you want clientset but why don't we add one more label to cache? then we can use client, no?

Reply:
Kserve previously removed cache for configmap because of memory leak issue and I didn't want to add it back.